reporter/src/main/kotlin/reporters/WebAppReporter.kt

```diff
@@ -55,7 +57,10 @@ class WebAppReporter : Reporter {
         FileOutputStream(outputFile, /* append = */ true).use { outputStream ->
             val b64OutputStream = Base64.getEncoder().wrap(outputStream)
 
-            GzipCompressorOutputStream(b64OutputStream).bufferedWriter().use { gzipWriter ->
+            val gzipParameters = GzipParameters().apply {
+                compressionLevel = Deflater.BEST_COMPRESSION
+            }
+            GzipCompressorOutputStream(b64OutputStream, gzipParameters).bufferedWriter().use { gzipWriter ->
```

Comment:
Nit: Blank line before this one. (Actually, I'd rather drop the blank line before line 60 if you want more compact code.)

Reply:
Merged, because I like the linebreaks the way they are.